doc/instances.md

```diff
@@ -46,6 +46,10 @@ limits.cpu                                  | string    | - (all)           | ye
 limits.cpu.allowance                        | string    | 100%              | yes           | -                 | How much of the CPU can be used. Can be a percentage (e.g. 50%) for a soft limit or hard a chunk of time (25ms/100ms)
 limits.cpu.priority                         | integer   | 10 (maximum)      | yes           | -                 | CPU scheduling priority compared to other instances sharing the same CPUs (overcommit) (integer between 0 and 10)
 limits.disk.priority                        | integer   | 5 (medium)        | yes           | -                 | When under load, how much priority to give to the instance's I/O requests (integer between 0 and 10)
+limits.hugepages.64KB                       | string    | -                 | yes           | container         | Fixed value in bytes (various suffixes supported, see below) to limit number of 64 KB hugepages
+limits.hugepages.1MB                        | string    | -                 | yes           | container         | Fixed value in bytes (various suffixes supported, see below) to limit number of 1 MB hugepages
+limits.hugepages.2MB                        | string    | -                 | yes           | container         | Fixed value in bytes (various suffixes supported, see below) to limit number of 2 MB hugepages
+limits.hugepages.1GB                        | string    | -                 | yes           | container         | Fixed value in bytes (various suffixes supported, see below) to limit number of 1 GB hugepages
```

Comment:
We still need a note somewhere that the availability of the various sizes differs based on architecture.